### DISCUSSION TITLE
How can I increase Git Clone Traffic date range in repository session insight?

### DISCUSSION THREAD

**Discussion by lypeInvictvs** [community/community#43575] [New to GitHub] [answered]
Hey Folks,

I would like to increase the interval between dates in the graph that shows the number of Git Clones in my repository. Is there any way to do this?

Thanks for any input!

This is an example:

<a href="https://imgur.com/3cr8Cp0.png">
  <img src="https://imgur.com/3cr8Cp0.png" />
</a>

**Comment by auchynnikau** [comment, marked as answer]
You can change the date range of the clone traffic data in a "Repository sessions" tab, there will be "Date range" dropdown located at the top of the page. But this page is only available for repositories that have opted into the GitHub Premium product. If you don't have a GitHub Premium subscription, you won't be able to access the repository session insight page.

Keep in mind that free options only provide clone traffic data for the past 14 days, and don't allow you to change the date range.

**Reply by kaczafakap** [reply]
00486052226669 call if U nred anserrs
